fuzzy/fsm_batch.go

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) HashiCorp, Inc.
 // SPDX-License-Identifier: MPL-2.0
 
+//go:build batchtest
```

Comment:
What is the minimum version of Go that supports this syntax? I suggest we bump go.mod to that version to avoid surprises.

Reply:
Addressed here: https://github.com/hashicorp/raft/pull/562/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6

Good call!